src/commands/bundle.zsh

```diff
@@ -9,7 +9,7 @@ antigen-bundle () {
     return 1
   fi
 
-  builtin typeset -A bundle; -antigen-parse-args 'bundle' ${=@}
+  builtin typeset -A bundle; -antigen-parse-args 'bundle' $@
```

Comment:
This causes issues with themes:

```
Installing desyncr/void void btype... Error! Activate logging and try again.
$ 

```

As `antigen-theme` calls `antigen-bundle` with `--btype=theme`.